lib/TransactionRunner.js

```diff
@@ -370,21 +375,21 @@ class TransactionRunner {
     } else {
       results = transaction.test.results = transaction.results;
```

Comment:
ditto, this should get updated. The added lines below are surrogate to a missing change on this line.

Reply:
Sorry, could you elaborate more on this?